tools/justfile

```diff
@@ -0,0 +1,136 @@
+# Enable positional args
+set positional-arguments
+
+# Default values for the deployment
+tag    := "test"
+queue  := "active"
+worker := "htcmock"
+
+# Base compose file
+export COMPOSE_BASE := "../docker-compose/docker-compose.yml"
+
+# loggin level of deployment
+export LOGGING_LEVEL := "Information"
+
+# Sets the queue
+export QUEUE := if queue == "rabbit" {
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+} else if queue == "rabbit091" {
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+} else if queue == "artemis" {
+  "../docker-compose/docker-compose.queue-artemis.yml"
+} else {
+  "../docker-compose/docker-compose.queue-activemqp.yml"
+}
+
+# Sets the override to feed docker-compose
+export OVERRIDE := if queue == "rabbit091" {
+  "../docker-compose/docker-compose.override-rabbitmq091.yml"
+} else {
+  "../docker-compose/docker-compose.override.yml"
+}
+
+# Defines worker and enviroment variables for deployment
+defaultWorkerImage := if worker == "stream" {
+  "dockerhubaneo/armonik_core_stream_test_worker:" + tag
+} else {
+  "dockerhubaneo/armonik_core_htcmock_test_worker:" + tag
+}
+defaultWorkerDockerFile := if worker == "stream" {
+  "../Tests/Stream/Server/Dockerfile"
+} else {
+  "../Tests/HtcMock/Server/src/Dockerfile"
+}
+export ARMONIK_WORKER             := env_var_or_default('WORKER_IMAGE', defaultWorkerImage)
+export ARMONIK_WORKER_DOCKER_FILE := env_var_or_default('WORKER_DOCKER_FILE', defaultWorkerDockerFile)
+export ARMONIK_METRICS            := "dockerhubaneo/armonik_control_metrics:" + tag
+export ARMONIK_PARTITIONMETRICS   := "dockerhubaneo/armonik_control_partitionmetrics:" + tag
+export ARMONIK_SUBMITTER          := "dockerhubaneo/armonik_control:" + tag
+export ARMONIK_POLLINGAGENT       := "dockerhubaneo/armonik_pollingagent:" + tag
+
+# List recipes and their usage
+@default:
+  just --list
+  just _usage
+  echo
+
+_usage:
+  #!/usr/bin/env bash
+  set -euo pipefail
+  echo
+  echo "The recipe deploy uses three variables"
+  echo
+  echo "usage: just tag=<tag> queue=<queue> worker=<worker> deploy"
+  echo "       if any of the variables is not set, its default value is used"
+  echo
+  echo "   tag: The core tag image to use, defaults to test"
+  echo
+  echo "   queue: allowed values below"
+  cat <<-EOF
```

Comment:
Either all `echo`, or a single `cat <<-EOF``, but the mixing of both is a bit weird